libr/io/plugin.c

```diff
@@ -100,28 +103,55 @@ R_API int r_io_plugin_list(RIO *io) {
 R_API int r_io_plugin_list_json(RIO *io) {
 	RIOPlugin *plugin;
 	SdbListIter *iter;
+	PJ *pj = pj_new ();
+	if (!pj) {
+		return NULL;
+	}
+	
 	char str[4];
 	int n = 0;
-	io->cb_printf("{\"IO_Plugins\":[");
+	pj_o (pj);
+	pj_k (pj, "IO_Plugins");
```

Comment:
i dont like this camel snake case. maybe better to just use "io_plugins"

Reply:
I changed it.